depends/packages/qt.mk

```
@@ -103,11 +102,11 @@ $(package)_config_opts_linux += -system-freetype
 $(package)_config_opts_linux += -no-feature-sessionmanager
 $(package)_config_opts_linux += -fontconfig
 $(package)_config_opts_linux += -no-opengl
-$(package)_config_opts_arm_linux += -platform linux-g++ -xplatform wagerr-linux-g++
+$(package)_config_opts_arm_linux += -platform linux-g++ -xplatform pivx-linux-g++
```

Fix naming back to `wagerr-*`.

```suggestion
$(package)_config_opts_arm_linux += -platform linux-g++ -xplatform wagerr-linux-g++
```